src/NimBLEDevice.cpp

```diff
@@ -567,10 +567,11 @@ int NimBLEDevice::getNumBonds() {
 
 /**
  * @brief Deletes all bonding information.
+ * @returns 0 on success, non-zero on failure.
```

Comment:
To be consistent with the rest of the code we should just return true/false. The error code can be printed in a log message for debugging.